src/core/trulens/core/database/migrations/versions/6_populate_app_name_and_version_data.py

```diff
@@ -34,7 +34,8 @@ def upgrade(config) -> None:
     orm = make_orm_for_prefix(table_prefix=prefix)
     apps = session.query(orm.AppDefinition).all()
     for app in apps:
-        app.app_version = "latest"
+        app.app_name = "default_app"
```

Comment:
wait, shouldn't we do this only if `app_name` isn't set? And for the line below, shouldn't we do it only if `app_version` isn't set?